hugegraph-core/src/main/java/org/apache/hugegraph/StandardHugeGraph.java

```diff
@@ -215,6 +212,8 @@ public StandardHugeGraph(HugeConfig config) {
 
         try {
             this.tx = new TinkerPopTransaction(this);
+            boolean supportsPersistence = this.backendStoreFeatures().supportsPersistence();
+            this.features = new HugeFeatures(this, supportsPersistence);
```

Comment:
seems the ci fails due to this.backendStoreFeatures() call, try to move to line 222?

Reply:
here are backendStoreFeatures() call tx.readwrite cause ConnectionException, but Open method catch the exception, throw RuntimeException, so here need change the unit test